### PR TITLE
Chore insights overview layout

### PR DESCRIPTION
**Pull request by hpneo**

### CODE REVIEW

File: src/components/App/HeaderAndSidebar/HeaderAndSidebar.js [hunk 1]
```diff
@@ -15,6 +15,7 @@ import Content from "components/layout/Content";
 import SearchBox from "components/shared/SearchBox";
 import SidebarLink from "components/layout/SidebarLink";
 import HeaderProfile from "components/layout/HeaderProfile";
+import styles from "./HeaderAndSidebar.module.css";
```
Comment (alessandrapereyra) [inline src/components/App/HeaderAndSidebar/HeaderAndSidebar.js:18]: Just of curiosity as I haven't been working on this before, as we going with `HeaderAndSidebar.module.css` vs `styles.module.css` as a standard?

File: src/components/InsightsPage/InsightsPage.js [hunk 2]
```diff
@@ -16,19 +18,19 @@ const InsightsPage = () => (
       alignment="center"
       margin="0 0 16px 0"
     >
-      <Heading as="h1" margin="0 0 4px 0">
+      <Heading as="h1" margin={0}>
```
Comment (alessandrapereyra) [inline src/components/InsightsPage/InsightsPage.js:21]: Can't this be just "margin='0'"?

File: src/components/InsightsPage/InsightsPage.js [hunk 3]
```diff
@@ -55,12 +58,43 @@ const InsightsPage = () => (
...
               pointHoverBorderWidth: 10,
-              data: [4, 5, 4, 6, 11, 9, 15],
+              data: [4, 5, 3, 6, 11, 9, 15],
               fill: "start",
             },
           ],
         }}
+        margin="0px 0 0px 24px"
       />
+      <Grid cols={2} rows={2} className={styles.summary} margin="16px 0 0 0">
```
Comment (alessandrapereyra) [inline src/components/InsightsPage/InsightsPage.js:68]: Wondering if we need the {} for those items in cols/rows.

File: src/components/App/HeaderAndSidebar/HeaderAndSidebar.js (hunk 1 above)
Comment (hpneo) [reply src/components/App/HeaderAndSidebar/HeaderAndSidebar.js:18]: Not sure, I was following @raffo1234's naming for other components.

**Pull request merged**